src/Microsoft.OData.Core/ODataBatchWriter.cs

```diff
@@ -82,28 +60,23 @@ public sealed class ODataBatchWriter : IODataBatchOperationListener, IODataOutpu
         /// <summary>
         /// Constructor.
         /// </summary>
-        /// <param name="rawOutputContext">The output context to write to.</param>
-        /// <param name="batchBoundary">The boundary string for the batch structure itself.</param>
-        internal ODataBatchWriter(ODataRawOutputContext rawOutputContext, string batchBoundary)
+        /// <param name="outputContext">The output context to write to.</param>
+        internal ODataBatchWriter(ODataRawOutputContext outputContext)

```

Comment:
>outputContext [](start = 56, length = 13)

why do you need to change variable name here?

Reply:
Actually the type should be changed to ODataOutputContext. Will fix.